fetch/metadata/redirect/redirect-http-upgrade.sub.html

```diff
@@ -6,62 +6,12 @@
 <script src=/fetch/metadata/resources/redirectTestHelper.sub.js></script>
 <script src=/common/security-features/resources/common.sub.js></script>
 <script src=/common/utils.js></script>
-<style>
-  @font-face {
-    font-family: myUpgradedFont;
-    src: url(http://{{host}}:{{ports[http][0]}}/fetch/api/resources/redirect.py?location=https%3A%2F%2F{{host}}%3A{{ports[https][0]}}%2Ffetch%2Fmetadata%2Fresources%2Frecord-header.py%3Ffile%3Dfont-https-upgrade{{$id:uuid()}});
-  }
-  #fontTest {
-    font-family: myUpgradedFont;
-  }
-</style>
 <body>
-    <div id="fontTest">Upgraded font</div>
     <script>
   let nonce = "{{$id}}";
```

Comment:
This is actually broken, as the `{{$id:uuid()}}` call on 12 is gone. But also: we don't need this line anymore. :)